src/instructlab/model/download.py

```diff
@@ -220,26 +214,20 @@ def download(self):
             f"oci:{oci_dir}",
             "--remove-signatures",
         ]
-        if self.ctx.obj is not None and logger.isEnabledFor(logging.DEBUG):
+        if self.log_level == "DEBUG" and logger.isEnabledFor(logging.DEBUG):
```

Comment:
What's the difference between these two checks?

Reply:
we were only using the ctx obj to access the log level. Once this functionality is behind a server layer there should not an expectation of a ctx obj being passed here because other clients may not have that notion

Reply:
first check verifies if the client asked for debug level and the second check verifies if the logger instance is actually configured to handle debug or not

Reply:
ack, makes sense